sql/src/main/java/io/crate/planner/operators/LogicalPlan.java

```diff
@@ -100,6 +101,15 @@
          *                       Collect (usedColumns: [a] - inherited from Order)
          *                         outputs: [_fetch, a]
          *                    </pre>
+         *
+         *                    These plans might later be optimized, see #{@link LogicalPlan#tryOptimize(LogicalPlan)},
```

Comment:
@mfussenegger Maybe better suggestion to phrase this?

Reply:
I think the phrasing is ok.